src/staking-v3/components/PeriodStats.vue

```diff
@@ -99,7 +99,7 @@ export default defineComponent({
         .map((x) => ({
           name: x.name,
           iconUrl: x.iconUrl,
-          amount: x.rewardAmount,
+          amount: BigInt(x.rewardAmount),
```

Comment:
```
export type PeriodData = {
  dappAddress: string;
  rewardAmount: bigint;
  stakeAmount: bigint;
};
```

I think it's already bigint

Reply:
Me too, but it is treated as string, most likely the value comes as string from API. I will check and cast immediately after fetching.

Reply:
Fixed in https://github.com/AstarNetwork/astar-apps/pull/1317/commits/cdf64d0e6c25fc2a354ec58797e83e6290026ac4 
coverting to big int as soon as data is received

Reply:
ah I see, thanks!